lib/mumuki/laboratory/locales/en.yml

```diff
@@ -11,88 +13,112 @@ en:
+    terms of <a href="https://creativecommons.org/licenses/by-sa/4.0/" target="_blank">Creative
+    Commons License Share-Alike, 4.0</a>
+  authoring_note_with_collaborators_html: This guide's content was developed by %{authors}
+    and <a href="%{collaborators}" target="_blank">many others</a>, under the terms
+    of <a href="https://creativecommons.org/licenses/by-sa/4.0/" target="_blank">Creative
+    Commons License Share-Alike, 4.0</a>
   back_to_mumuki: Back to Mumuki!
   bibliotheca_ui: Bibliotheca
   birthdate: Birthdate
   blocked_forum_explanation: Are you in the middle of an exam right now?
   cancel: Cancel
   cancel_subscription: Cancel your subscription.
   chapter: Chapter
-  chapters: Chapters
   chapter_finished_html: You have finished %{chapter}!
   chapter_number: Chapter %{number}
+  chapters: Chapters
   choose_your_avatar: Choose your avatar
   classroom_ui: Classroom
   clear_console: Clear console
   closed: Closed
-  closed_count: '%{count} closed'
-  created_at_asc: Oldest
-  created_at_desc: Newest
+  closed_count: "%{count} closed"
   comment: Comment
   confirm: Confirm
   confirm_reset: You are about to restart your exercise. Do you want to proceed?
-  confirm_restart: You are about to delete your progress for this guide. Do you want to proceed?
+  confirm_restart: You are about to delete your progress for this guide. Do you want
+    to proceed?
   console: Console
-  contact_administrator: 'If you think this is not supposed to happen, please contact %{link}'
+  contact_administrator: If you think this is not supposed to happen, please contact
+    %{link}
   content: Content
-  continue_exercise: "Continue"
+  continue_exercise: Continue
   continue_lesson: Continue this lesson!
   continue_practicing: Continue practicing!
   corollary: To think about
   correct_answer: The answer is correct!
   create_submission: Submit
   created_at: Created at
+  created_at_asc: Oldest
+  created_at_desc: Newest
   date: Date
   description: Description
-  destroy_message: delete the message
   destroy: Destroy
-  disabled_explanation: You are trying to visit a permamently disabled or deleted resource
+  destroy_message: delete the message
+  disabled_explanation: You are trying to visit a permamently disabled or deleted
+    resource
   disabled_organization_explanation: This path has already finished.
-  discussion_description_placeholder: You can add more information regarding your doubt.
+  discussion_description_placeholder: You can add more information regarding your
+    doubt.
+  discussion_new_message: New message on %{title}
   discussion_updated: Discussion updated
   discussions: Discussions
   dont_leave_us: Don't leave us! Learning is fun. You just have to keep at it.
   download: Download your solution
   edit: Edit
   edit_profile: Edit profile
-  editor_placeholder: "write your solution here..."
+  editor_placeholder: write your solution here...
+  eeee: s
```

Comment:
We might not be using this one anytime soon :stuck_out_tongue:

Reply:
Seems that you don't understand the "eeee" language... 🤣